music/src/core/player.ts

```diff
@@ -475,9 +480,10 @@ export class SoundFontPlayer extends BasePlayer {
 
   constructor(
       soundFontURL: string, output = Tone.Master,
-      programOutputs?: Map<number, any>,      // tslint:disable-line:no-any
-      drumOutputs?: Map<number, any>,         // tslint:disable-line:no-any
-      callbackObject?: BasePlayerCallback) {  // tslint:disable-line:no-any
+      programOutputs?: Map<number, any>,  // tslint:disable-line:no-any
+      drumOutputs?: Map<number, any>,     // tslint:disable-line:no-any
+      callbackObject?: BasePlayerCallback) {
+    // tslint:disable-line:no-any
```

Comment:
i don't think this last comment is needed?

Reply:
oh weird. this is clang doing some weird formatting but yeah it wasn't needed in the first place